docs/technical_reference/unit_models/translators/translator_asm2d_adm1.rst

```diff
@@ -150,8 +150,8 @@ Equations and Relationships (With Decay)
    "S_K concentration", ":math:`S_{K, 6} = S_{K, in} + (K_{XPP} * X_{PP, in})`"
    "S_Mg concentration", ":math:`S_{Mg, 6} = S_{Mg, in} + (Mg_{XPP} * X_{PP, in})`"
 
-Equations and Relationships (Without Decay)
--------------------------------------------
+Equations and Relationships (with phosphorus biomass handled implicitly)
```

Comment:
One of these should be labeled `without biological phosphorus removal` and the other `with`